controller-server/src/test/java/com/yahoo/vespa/hosted/controller/versions/VersionStatusTest.java

```diff
@@ -81,6 +81,8 @@ public void testVersionStatusAfterApplicationUpdates() {
 
         // version2 is released
         tester.upgradeSystem(version2);
+        tester.readyJobTrigger().maintain();
```

Comment:
Consider adding a helper method to the tester, e.g. `DeploymentTester.triggerReadyJobs(int times)`.

Reply:
Yes :)